packages/react-reconciler/src/ReactChildFiber.new.js

```diff
@@ -46,7 +46,7 @@ import {
 } from './ReactFiber.new';
 import {emptyRefsObject} from './ReactFiberClassComponent.new';
 import {isCompatibleFamilyForHotReloading} from './ReactFiberHotReloading.new';
-import {StrictMode} from './ReactTypeOfMode';
+import {StrictLegacyMode} from './ReactTypeOfMode';
```

Comment:
Chatted offline and we can follow up with this if we agree - we may want to name this something other than "legacy" since the term overloaded to mean "legacy root", "legacy strict mode" and any other "legacys" we have. I was thinking something like:

- `NoMode` = 0
- `StrictWarningMode` = Level 1
- `StrictRenderMode` = Level 2
- `StrictEffectMode` = Level 3